torch/_inductor/cpp_builder.py

```diff
@@ -320,26 +320,14 @@ def _get_optimization_cflags() -> List[str]:
         if not config.cpp.enable_floating_point_contract_flag:
             cflags.append("ffp-contract=off")
 
-        if config.is_fbcode():
-            # FIXME: passing `-fopenmp` adds libgomp.so to the generated shared library's dependencies.
-            # This causes `ldopen` to fail in fbcode, because libgomp does not exist in the default paths.
-            # We will fix it later by exposing the lib path.
-            return cflags
-
-        if sys.platform == "darwin":
-            # Per https://mac.r-project.org/openmp/ right way to pass `openmp` flags to MacOS is via `-Xclang`
-            # Also, `-march=native` is unrecognized option on M1
-            cflags.append("Xclang")
-        else:
+        if sys.platform != "darwin":
+            # https://stackoverflow.com/questions/65966969/why-does-march-native-not-work-on-apple-m1
+            # `-march=native` is unrecognized option on M1
```

Comment:
This comment does not make sense here, as check for `sys.platform != "darwin" (and this will regress windows behavior, wouldn't it?)

Reply:
I will update the code.

Reply:
@malfet The code checked `Windows` at upper block:
<img width="609" alt="image" src="https://github.com/pytorch/pytorch/assets/8433590/a13dac76-e700-489a-8621-773dbcd282b7">